draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -188,7 +207,8 @@ Verifier:
            +------------+
                  |
            Presents SD-JWT
```

Comment:
this read kinda awkward to me and I think things can be general in a flow diagram like this 
```suggestion
           Presents SD-JWT-KB
```

Reply:
wait, what the... why can't I commit this suggestion anymore?

Reply:
I don't know how to solve it, just know it happened to me before too...

Reply:
I *think* what happened is that I committed a different related nearby suggestion (that the UI wouldn't let me do in one shot) that basically put the PR into a state of having no changes near here and thus talking this area out of the scope of suggestible changes that github will let you make in the UI. If that jumble of words even makes any sense. I "solved" it by making the changes locally and pushing to Richard's branch with 2db05069e9d8cb749142a30a4285a416fbdfc88a and then ea2500ec6ac09aba9c2988078df708f1fb1369a8 which is probably a breach of some etiquette but was the best I could manage as the situation unfolded.